xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/api/Property.java

```diff
@@ -82,18 +85,31 @@ public PropertyClass getPropertyClass()
      */
     public java.lang.Object getValue()
     {
-        BaseProperty baseProperty = getBaseProperty();
+        final BaseProperty<?> baseProperty = getBaseProperty();
+        final XWikiContext context = getXWikiContext();
```

Comment:
introduced local variable to escape checkstyle "line too long" warning.

Reply:
Note that we usually don't use `final` keyword.